tests/queries/0_stateless/02907_http_exception_json_bug.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+CUR_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CUR_DIR"/../shell_config.sh
+
+CH_URL="$CLICKHOUSE_URL&http_write_exception_in_output_format=1&allow_experimental_analyzer=0"
```

Comment:
```suggestion
CH_URL="$CLICKHOUSE_URL&http_write_exception_in_output_format=1"
```

Extra `allow_experimental_analyzer=0` shouldn't be necessary